background.html

```diff
@@ -100,6 +100,7 @@ <h3>{{ welcomeToSignal }}</h3>
         </div>
       </div>
     </div>
+    <div class='lightboxContainer'></div>
```

Comment:
Sure, I can do that for now. Could we revisit this later and consider doing `camelCase` and `PascalCase` to match component names and make it easier for CSS-in-JS to avoid quoting everything, e.g. `{lightboxContainer: true}` vs `{'lightbox-container': true}`? It’s also nice to use find / replace to search for related things if they don’t differ due to `camelCase` vs `dash-case`.